borgbase_repo.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/python
+
+from pprint import pprint
```

Comment:
unused import

Reply:
Good point, I use this during the development phase to be able to print out variables. I'll remove it.